RELEASE_NOTES.md

```diff
@@ -97,6 +97,8 @@
 - Fixed a crash validating --device argument for create and run ([#19335](https://github.com/containers/podman/issues/19335)).
 - Fixed a bug where `.HostConfig.PublishAllPorts` always evaluates to `false` when inspecting a container created with `--publish-all`.
 - Fixed a bug in `podman image trust` command to allow using the local policy.json file ([#19073](https://github.com/containers/podman/issues/19073)).
+- Fixed a bug where a cgroup was not correctly mounted when running without a network namespace in rootless mode ([#20073](https://github.com/containers/podman/issues/20073)).
```

Comment:
I'd phrase it as "Fixed a bug where the cgroup file system was not correctly mounted inside the container when running without a network namespace in rootless mode"

Reply:
done